pkg/ormb/ormb.go

```diff
@@ -0,0 +1,43 @@
+package ormb
+
+import (
+	"github.com/spf13/viper"
+
+	"github.com/kleveross/klever-model-registry/pkg/common"
+	"github.com/kleveross/klever-model-registry/pkg/util"
+)
+
+type ORMBClient struct {
+}
+
+func New() Interface {
+	return &ORMBClient{}
+}
+
+func (s *ORMBClient) ORMBLogin() error {
+	ormbDomain := viper.GetString(common.ORMBDomainEnvKey)
+	ormbUserName := viper.GetString(common.ORMBUsernameEnvkey)
+	ormbPassword := viper.GetString(common.ORMBPasswordEnvKey)
+
+	return util.ExecOSCommand("ormb", []string{"login", ormbDomain, "--insecure", "-u", ormbUserName, "-p", ormbPassword})
```

Comment:
Why not using ormb as a package?

Reply:
OK, I will update it by using ormb package, and ormb need mock pacakge, I will do it.

Reply:
Comment is addressed, PTAL